recipe/conda_build_config.yaml

```diff
@@ -1,8 +1,20 @@
-c_compiler_version:        # [linux and not aarch64]
-  - 8                      # [linux and not (s390x or aarch64 or ppc64le)]
-  - 8                      # [linux and ppc64le]
-  - 8                      # [linux and s390x]
-cxx_compiler_version:      # [linux]
-  - 8                      # [linux and not (s390x or aarch64 or ppc64le)]
-  - 8                      # [linux and ppc64le]
-  - 8                      # [linux and s390x]
+c_compiler_version:         # [linux and not aarch64]
+  - 8                       # [linux and not (s390x or aarch64 or ppc64le)]
+  - 8                       # [linux and ppc64le]
+  - 8                       # [linux and s390x]
+cxx_compiler_version:       # [linux]
+  - 8                       # [linux and not (s390x or aarch64 or ppc64le)]
+  - 8                       # [linux and ppc64le]
+  - 8                       # [linux and s390x]
+
+cxx_compiler:               # [win]
+  - 'vs2019'                # [win]
+c_compiler:                 # [win]
+  - 'vs2019'                # [win]
+vc:                         # [win]
+  - 16                      # [win]
+
+CONDA_BUILD_SYSROOT:       # [osx and x86]
+  - /opt/MacOSX10.13.sdk   # [osx and x86]
+macos_min_version:         # [osx and x86]
```

Comment:
@boldorider4 if you wonder about the solution for the problem we talked about, here it is